Tests/UnitTests/Core/Detector/IndexedSurfacesGeneratorTests.cpp

```diff
@@ -75,8 +77,12 @@ BOOST_AUTO_TEST_CASE(RingDisc1D) {
   // Check that surfaces 10, 11, 12 build the bins at phi == 0
   std::vector<size_t> reference = {10, 11, 12};
   GridType::point_t p = {0.05};
+
+  std::cout << "here ok" << std::endl;
   BOOST_CHECK(grid.atPosition(p) == reference);
 
+  std::cout << "here ok" << std::endl;
+
```

Comment:
Could you maybe put a more meaningfull text into the `cout`s (if needed at all)? :)

Reply:
> Could you maybe put a more meaningfull text into the `cout`s (if needed at all)? :)
oh sorry, I forgot to remove it! Thanks!